examples/contour/03-contour.yaml

```diff
@@ -55,6 +55,9 @@ spec:
           name: xds
           protocol: TCP
         - containerPort: 8000
+          name: metrics
+          protocol: TCP
+        - containerPort: 6060
```

Comment:
I don't see a need to add the debug port to the deployment. By default it only listens on `127.0.0.1:6060` you can't get to it unless you port-forward to the pod.

Reply:
@stevesloka `containerPort` is informational only. Container network ports will only be exposed by the containerized app. I including the debug port here so users understand the app is listening on 6060  (even though it's local host). I can remove if you prefer.

Reply:
I would remove since it feels weird to me to have a debug port on a production system, but chatting with some others it's fine to leave.